gensim/models/keyedvectors.py

```diff
@@ -839,6 +974,11 @@ def most_similar_cosmul(self, positive=None, negative=None, topn=10):
             List of words that contribute negatively.
         topn : int, optional
             Number of top-N similar words to return.
+        restrict_vocab : int, optional
+            Optional integer which limits the range of vectors which
+            are searched for most-similar values. For example, restrict_vocab=10000 would
+            only check the first 10000 word vectors in the vocabulary order. (This may be
+            meaningful if you've sorted the vocabulary by descending frequency.)
```

Comment:
How would a user `sort the vocabulary by descending order`? Let's be more concrete.

Reply:
An example would be really helpful here.